src/vcpkg/commands.version.cpp

```diff
@@ -9,9 +9,9 @@
 namespace
 {
     using namespace vcpkg;
-    constexpr StringLiteral version_init = VCPKG_BASE_VERSION_AS_STRING "-" VCPKG_VERSION_AS_STRING
+    static constexpr StringLiteral version_init = VCPKG_BASE_VERSION_AS_STRING "-" VCPKG_VERSION_AS_STRING
```

Comment:
I think you should revert this change because it has no effect. (entities declared within anonymous namespaces already have internal linkage, see http://eel.is/c++draft/basic.link#4 )